mlflow/fastai.py

```diff
@@ -17,8 +17,6 @@
 import gorilla
 import tempfile
 import shutil
-import pandas as pd
```

Comment:
pandas and numpy dependencies in flavors should be lazy loaded https://github.com/mlflow/mlflow/issues/3577